docs/guide/core-concepts.rst

```diff
@@ -1152,77 +1327,18 @@ depending on your architecture.
-
-A Layout Definition describes the location, dimensions of, and
-organization of a tiled area of a map. Conceptually, the tiled area
-forms a grid, and the Layout Definitions describes that grid's area and
-cell width/height. These definitions can be used to chop a bundle of
-imagery into tiles suitable for being served out on a web map.
-
-Within the context of GeoTrellis, the ``LayoutDefinition`` class extends
-``GridExtent``, and exposes methods for querying the sizes of the grid
-and grid cells. Those values are stored in the ``TileLayout`` (the grid
-description) and ``CellSize`` classes respectively.
-``LayoutDefinition``\ s are used heavily during the raster reprojection
-process. Within the context of Geotrellis, the ``LayoutDefinition``
-class extends ``GridExtent``, and exposes methods for querying the sizes
-of the grid and grid cells. Those values are stored in the
-``TileLayout`` (the grid description) and ``CellSize`` classes
-respectively. ``LayoutDefinition``\ s are used heavily during the raster
-reprojection process.
-
-**What is a Layout Scheme?**
-
-The language here can be vexing, but a ``LayoutScheme`` can be thought
-of as a factory which produces ``LayoutDefinition``\ s. It is the scheme
-according to which some layout definition must be defined - a layout
-definition definition, if you will. The most commonly used
-``LayoutScheme`` is the ``ZoomedLayoutScheme``, which provides the
-ability to generate ``LayoutDefinitions`` for the different zoom levels
-of a web-based map (e.g. `Leaflet <http://leafletjs.com>`__).
-
-| **How are Layout Definitions used throughout Geotrellis?**
-| Suppose that we've got a distributed collection of
-  ``ProjectedExtent``\ s and ``Tile``\ s which cover some contiguous
-  area but which were derived from GeoTIFFs of varying sizes. We will
-  sometimes describe operations like this as 'tiling'. The method which
-  tiles a collection of imagery provided a ``LayoutDefinition``, the
-  underlying ``CellType`` of the produced tiles, and the
-  ``ResampleMethod`` to use for generating data at new resolutions is
-  ``tileToLayout``. Let's take a look at its use:
 
-.. code-block:: scala
-
-    val sourceTiles: RDD[(ProjectedExtent, Tile)] = ??? // Tiles from GeoTIFF
-    val cellType: CellType = IntCellType
-    val layout: LayoutDefinition = ???
-    val resamp: ResampleMethod = NearestNeighbor
-
-    val tiled: RDD[(SpatialKey, Tile)] =
-      tiles.tileToLayout[SpatialKey](cellType, layout, resamp)
-
-In essence, a ``LayoutDefinition`` is the minimum information required
-to describe the tiling of some map's area in Geotrellis. The
-``LayoutDefinition`` class extends ``GridExtent``, and exposes methods
-for querying the sizes of the grid and grid cells. Those values are
-stored in the ``TileLayout`` (the grid description) and ``CellSize``
-classes respectively. ``LayoutDefinition``\ s are most often encountered
-in raster reprojection processes.
 
 Map Algebra
```

Comment:
Same as above comment.

Reply:
In this case, I feel like map algebra isn't fundamental enough or unique enough to GT that I want to clutter up the glossary, which should ultimately be a very quick way to get used to our terminology.

Reply:
That's true. I can see how map algebra isn't that crucial or unique enough to list.